composer.json

```diff
@@ -22,7 +22,11 @@
 		"source": "https://github.com/ICanBoogie/Storage"
 	},
 	"require": {
-		"php": ">=5.5.0"
+		"php": ">=7.1"
+	},
+	"require-dev": {
+		"ext-json": "*",
+		"ext-apcu": "*"
```

Comment:
Will installation fail if i don't have `ext-apcu`?

Reply:
The requirement only applies to local development, to be able to run the test suite. The extension is not required to use the library. `require-dev` is root only, it only applies to the root package context.

https://getcomposer.org/doc/04-schema.md#require-dev

Reply:
Ah... yep, makes sense.